lib/ansible/modules/clustering/kubevirt/kubevirt_vm_status.py

```diff
@@ -1,184 +0,0 @@
-#!/usr/bin/python
-# -*- coding: utf-8 -*-
-
-# Copyright (c) 2018, KubeVirt Team <@kubevirt>
-# Apache License, Version 2.0
-# (see LICENSE or http://www.apache.org/licenses/LICENSE-2.0)
-
-ANSIBLE_METADATA = {'metadata_version': '1.1',
```

Comment:
I would keep this code so we could still support inline in case user want to use something which is not supported.